extensions/wikia/Recirculation/js/recirculation.js

```diff
@@ -62,143 +40,52 @@ require([
 					domainType: 'fandom.wikia.com'
 				}
 			},
-			wikiItems: {
-				max: $mixedContentFooter.data('number-of-wiki-articles'),
-				widget: 'wikia-footer-wiki-rec',
-				width: 386,
-				height: 337,
-				modelName: 'wiki',
-				opts: {
-					rule_language: window.wgContentLanguage
-				}
-			}
 		};
 
-	function prepareRailRecirculation(options) {
-		if (mw.config.get('showEmbeddedFeed')) {
```

Comment:
I think we'd like to have this condition fulfilled somehow, did you implement it in new approach? I can not find it